GPU/Workflow/src/GPUWorkflowITS.cxx

```diff
@@ -35,9 +39,21 @@ void GPURecoWorkflowSpec::initFunctionITS(o2::framework::InitContext& ic)
 {
   o2::its::VertexerTraits* vtxTraits = nullptr;
   o2::its::TrackerTraits* trkTraits = nullptr;
+#ifdef ENABLE_UPGRADES
```

Comment:
you can simplify this to 
```
#ifdef ENABLE_UPGRADES
if (mSpecConfig.isITS3) {
    mITSTrackingInterface = std::make_unique<o2::its3::ITS3TrackingInterface>(mSpecConfig.processMC,
                                                                              mSpecConfig.itsTriggerType,
                                                                              mSpecConfig.itsOverrBeamEst);
  } else
#endif
{
    mITSTrackingInterface = std::make_unique<o2::its::ITSTrackingInterface>(mSpecConfig.processMC,
                                                                            mSpecConfig.itsTriggerType,
                                                                            mSpecConfig.itsOverrBeamEst);
  }
```
to avoid the copy and paste.

Reply:
done.